data/campaigns/World_Conquest/lua/map/scenario_utils/plot.lua

```diff
@@ -49,7 +49,11 @@ function add_plot(scenario, scenario_num, nplayers)
 		end
 		start_message(ai1, true, _ "Foolish upstarts. You will never defeat us! Come, my allies. Let us crush them!")
 		start_message(ai1, false, _ "Yes! We will drive them into the sea!")
-		start_message("1,2,3", false, _ "Never fear! Your trusted friends stand beside you!")
+		if nplayers == 4 then
+			start_message("4", true, _ "Never fear! Your trusted friends stand beside you!")
```

Comment:
This seems weird. I would expect you'd just add ",4" to the first parameter. Now you either get a message from side 4 or from side 1, 2 and 3.

Also this changes a single `start_message` call when there are plenty more similar ones as well as `end_message`.